### ISSUE TITLE
touchscreen compatible diagram setup

### ISSUE THREAD

**Issue by dturevski** [wfcc/ankona#31]
Is it possible to add alternate simple point-and-click interface to diagram setup facilities?
The problem is that current drag-and-drop doesn't work with touchscreen devices (tested with Android and iOS)

**Comment by punund** [comment]
No, as this would require us to amend the Codex so that phones are banned during WCSC even in silent mode ;)

Seriously, this can be addressed, provided someone would give a hand with testing, specifically with Android, as I don't have any devices myself.

**Comment by dturevski** [comment]
Count me in for Android testing then :)
Actually, I think there should not be any complex issues if you stick to just onclick handling (i.e. no gestures)
